File: JavaScript solutions/9. Palindrome Number.js
/**
 * @param {number} x
 * @return {boolean}
 */
function reverseString(str) {
    return str.split("").reverse().join("");
}

var isPalindrome = function(x) {
    var str=x.toString();
    var rx=reverseString(str);
    if(rx==x) return true;
    return false;
};

File: C++ solutions/9. Palindrome Number.cpp
#include<bits/stdc++.h>
using namespace std;

// solution
class Solution {
public:
    bool isPalindrome(int x) {
        string s=to_string(x);
        string st=s;
        reverse(st.begin(),st.end());
        if(s==st) return true;
        else return false;
    }
};